### DISCUSSION TITLE
Not able to send emails

### DISCUSSION THREAD

**Discussion by ClaudeWithYouForever** [outline/outline#6646] [Self Hosting]
Hello ,everyone, i try to install the outline wiki  by using docker  in my company. but i can't send emails by configuring smtp emails.
docker version：outlinewiki/outline:0.75.2
smtp configs:
`SMTP_HOST=mail.xxx.org
SMTP_PORT=465
SMTP_USERNAME=my@xxx.org
SMTP_PASSWORD=xxxx
SMTP_FROM_EMAIL=my@xxx.org
SMTP_REPLY_EMAIL=my@xxx.org
SMTP_TLS_CIPHERS=SSLv2
SMTP_SECURE=true
`
logs from outline continer:
`{"log":"{\"error\":\"error:0A0000B9:SSL routines::no cipher match\",\"level\":\"error\",\"message\":\"Error processing task in EmailTask\",\"stack\":\"Error: error:0A0000B9:SSL routines::no cipher match\\n    at configSecureContext (node:internal/tls/secure-context:168:11)\\n    at Object.createSecureContext (node:_tls_common:116:3)\\n    at Object.connect (node:_tls_wrap:1763:48)\\n    at /opt/outline/node_modules/nodemailer/lib/smtp-connection/index.js:321:40\\n    at module.exports.resolveHostname (/opt/outline/node_modules/nodemailer/lib/shared/index.js:117:20)\\n    at SMTPConnection.connect (/opt/outline/node_modules/nodemailer/lib/smtp-connection/index.js:299:27)\\n    at /opt/outline/node_modules/nodemailer/lib/smtp-transport/index.js:264:24\\n    at Immediate.\u003canonymous\u003e (/opt/outline/node_modules/nodemailer/lib/smtp-transport/index.js:71:35)\\n    at process.processImmediate (node:internal/timers:478:21)\"}\n","stream":"stdout","time":"2024-03-06T09:52:56.072589705Z"}`

**Comment by tommoor** [comment]
There is no `SSLv2` cipher available, try removing this configuration or using `SSLv3`

**Reply by capriciousduck** [reply]
I get the following error when using SSLv3 cipher.

`
outline         | {"error":"583B5A3E3D7F0000:error:0A00010B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:354:\n","level":"error","message":"Error processing task in EmailTask","stack":"Error: 583B5A3E3D7F0000:error:0A00010B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:354:\n"}
`

**Comment by breakingflower** [comment]
I'm getting a similar situation: 

.env file: 
```
SMTP_HOST=smtp.office365.com
SMTP_PORT=587
SMTP_USERNAME=myuser@mydomain
SMTP_PASSWORD=mypassword
SMTP_FROM_EMAIL=myuser <myuser@mydomain>
SMTP_REPLY_EMAIL=myuser <myuser@mydomain>
SMTP_NAME=mydomain
SMTP_TLS_CIPHERS=SSLv3
SMTP_SECURE=true
```

Result:
```
outline-1  | {"label":"worker","level":"info","message":"EmailTask running","props":{"actorEmail":"myuser@mydomain","actorName":"myuser","name":"test","teamName":"my knowledge base","teamUrl":"https://mydomain","to":"invite@user.com"},"templateName":"InviteEmail"}
outline-1  | {"label":"email","level":"info","message":"Sending email \"myuser invited you to join my knowledge base workspace\" to invite@user.com"}
outline-1  | {"label":"http","level":"info","message":"  --> POST /api/users.resendInvite 200 66ms 39b"}
outline-1  | {"error":"B80C16E7ED7F0000:error:0A00010B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:354:\n","level":"error","message":"Error sending email to floris@remmen.be","stack":"Error: B80C16E7ED7F0000:error:0A00010B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:354:\n"}
outline-1  | {"error":"B80C16E7ED7F0000:error:0A00010B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:354:\n","level":"error","message":"Error processing task in EmailTask","stack":"Error: B80C16E7ED7F0000:error:0A00010B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:354:\n"}
```

Leaving out `SMTP_TLS_CIPHERS`, or trying alternative options `TLSv1.2` (as in docs), `TLSv1.3` yield the same results. I also have the same results when switching to a different SMTP host.

**Reply by tommoor** [reply]
Try removing `SMTP_TLS_CIPHERS` and setting `SMTP_SECURE=false`. This does not mean that the connection made would be insecure.

**Reply by breakingflower** [reply]
That does remove that message, but highlights an organisational setting in microsoft: 
` user is locked by your organization's security defaults policy`

This has nothing to do with outline. It works with a gmail account. Thanks! 

Same as [#2343](https://github.com/outline/outline/discussions/2343)
